apps/volontulo/templates/offers/offer_form.html

```diff
@@ -5,73 +5,105 @@
+                    </div>
+                </div>
             </div>
         </div>
-        <div class="form-group form-group-sm">
-            <label class="col-xs-2 control-label" for="{{ offer_form.description.id_for_label }}">Opis</label>
-            <div class="col-xs-10">
-                <textarea class="form-control" rows="3" name="description" id="{{ offer_form.description.id_for_label }}">{{ request.POST.description|default:offer.description }}</textarea>
+
+        <div class="row">
+            <div class="col-xs-12 {% if form.description.errors %}has-error{% endif %}">
+                {% if form.description.errors %}<div class="col-xs-12">{{ form.description.errors }}</div>{% endif %}
+                <div class="col-xs-12 form-group form-group-sm">
+                    <div class="col-xs-5"><label class="control-label" for="{{ form.description.id_for_label }}">Opis</label></div>
+                    <div class="col-xs-7"><textarea class="form-control" rows="3" name="description" id="{{ form.description.id_for_label }}">{{ request.POST.description|default:offer.description }}</textarea></div>
+                </div>
             </div>
         </div>
-        <div class="form-group form-group-sm">
-            <label class="col-xs-2 control-label" for="{{ offer_form.benefits.id_for_label }}">Korzyści</label>
-            <div class="col-xs-4">
-                <textarea class="form-control" rows="3" name="benefits" id="{{ offer_form.benefits.id_for_label }}">{{ request.POST.benefits|default:offer.benefits }}</textarea>
+
+        <div class="row">
+            <div class="col-xs-6 {% if form.benefits.errors %}has-error{% endif %}">
+                {% if form.benefits.errors %}<div class="col-xs-12">{{ form.benefits.errors }}</div>{% endif %}
+                <div class="col-xs-12 form-group form-group-sm">
+                    <div class="col-xs-5"><label class="control-label" for="{{ form.benefits.id_for_label }}">Korzyści</label></div>
+                    <div class="col-xs-7"><textarea class="form-control" rows="3" name="benefits" id="{{ form.benefits.id_for_label }}">{{ request.POST.benefits|default:offer.benefits }}</textarea></div>
+                </div>
             </div>
-            <label class="col-xs-2 control-label" for="{{ offer_form.requirements.id_for_label }}">Wymagania</label>
-            <div class="col-xs-4">
-                <textarea class="form-control" rows="3" name="requirements" id="{{ offer_form.requirements.id_for_label }}">{{ request.POST.requirements|default:offer.requirements }}</textarea>
+            <div class="col-xs-6 {% if form.requirements.errors %}has-error{% endif %}">
+                {% if form.requirements.errors %}<div class="col-xs-12">{{ form.requirements.errors }}</div>{% endif %}
```

Comment:
Yes, it's OK. I just only want to display one depends on class, splitting it into more lines would be problematic for reader because of multiline class attribute.